modules/images-configuration-file.adoc

```diff
@@ -51,11 +51,15 @@ status:
 <1> `Image`: Holds cluster-wide information about how to handle images. The canonical, and only valid name is `cluster`.
 <2> `allowedRegistriesForImport`: Limits the container image registries from which normal users may import images. Set this list to the registries that you trust to contain valid images, and that you want applications to be able to import from. Users with permission to create images or `ImageStreamMappings` from the API are not affected by this policy. Typically only cluster administrators have the appropriate permissions.
 <3> `additionalTrustedCA`: A reference to a config map containing additional certificate authorities (CA) that are trusted during image stream import, pod image pull, `openshift-image-registry` pullthrough, and builds. The namespace for this config map is `openshift-config`. The format of the config map is to use the registry hostname as the key, and the PEM certificate as the value, for each additional registry CA to trust.
-<4> `registrySources`: Contains configuration that determines how the container runtime should treat individual registries when accessing images for builds and pods. For instance, whether or not to allow insecure access. It does not contain configuration for the internal cluster registry. This example lists `allowedRegistries`, which defines the registries that are allowed to be used. One of the registries listed is insecure.
+<4> `registrySources`: Contains configuration that determines how the container runtime should treat individual registries when accessing images for builds and pods. For instance, whether or not to allow insecure access. It does not contain configuration for the internal cluster registry. This example lists `allowedRegistries`, which defines the registries that are allowed to be used. The insecury registry `insecure.com` is allowed. The `blockedRegistries` list allows you to block specific registries. Either the `allowedRegistries` parameter or the `blockedRegistries` parameter can be set, but not both.
```

Comment:
"The insecury registry `insecure.com` is allowed." should likely be "The insecure registry `insecure.com` is allowed."